<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tables_tab</class>
 <widget class="QWidget" name="tables_tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#top_widget {
background-color: rgb(54, 71, 79);
}
#header {
background-color: rgb(200, 200, 200);
}

.QRadioButton, .QPushButton {
	color: rgb(255, 255, 255);
	font: 700 9pt &quot;Segoe UI&quot;;
	border: null;
	padding: 10%;
}

#table {
background-color: rgb(241, 244, 249);
}

.QRadioButton:hover, .QPushButton:hover{
background-color: rgb(69, 75, 84);
}

#rigth_widget {
	background-color: rgb(39, 50, 56);
}

.QLabel {
	color: rgb(136, 139, 145);
	font: 700 9pt &quot;Segoe UI&quot;;
	padding: 10%;
}

#table &gt; .QHeaderView::section {
    background-color: rgb(239, 243, 246);
	color: rgb(86, 96, 113);
border: 0;
border-bottom: 1px solid rgb(203, 206, 208);
}
#table &gt; .QTableCornerButton::section{ 
    background-color: transparent;

}
#table::item:selected {
	background-color: rgb(184, 236, 230);
color: black;
}
.QRadioButton, .QPushButton, .QCheckBox {
	background-color: rgba(255, 255, 255, 0);
	color: rgb(255, 255, 255);
	font: 700 9pt &quot;Segoe UI&quot;;
	padding: 10%;
	border: null;
}

.QComboBox {
	background-color: #36474f;
	color: rgb(255, 255, 255);
	font: 700 9pt &quot;Segoe UI&quot;;
	padding: 10%;
	border: null;
}

.QRadioButton:hover, .QPushButton:hover, .QCheckBox:hover{
background-color: rgb(69, 75, 84);
}

.QRadioButton:checked{
background-color: rgb(38, 165, 154);
}

.QRadioButton::indicator, .QCheckBox::indicator{
width: 0;
}

.QLabel {
	color: rgb(136, 139, 145);
	font: 700 9pt &quot;Segoe UI&quot;;
	padding: 10%;
}

.QSpinBox, .QDateEdit, .QLineEdit, .QTextEdit {
background-color: rgb(54, 71, 79);
border: null;
padding: 5%;
border-color: rgb(200, 200, 200);
color: rgb(200, 200, 200);
}
    
    QScrollBar:vertical {
        background-color: rgb(39, 50, 56);
        width: 10px;
        margin: 0px;
    }
    
    QScrollBar::handle:vertical {
        background-color: #c0c0c0;
        min-height: 20px;
    }
    
    QScrollBar::add-line:vertical,
    QScrollBar::sub-line:vertical {
        background: none;
    }
    
    QScrollBar::add-page:vertical,
    QScrollBar::sub-page:vertical {
        background: none;
    }

#r_custom_overlap:checked +#i_overlap_count {
background-color: rgb(38, 165, 154);
}


#r_custom_overlap:hover +#i_overlap_count {
background-color: rgb(38, 165, 154);
}

.QGroupBox {
border: null;
}

.QComboBox::drop-down {
width: 0;
}

QMenu::itemQComboBox QAbstractItemView{
color: white;
            padding: 5px 20px;
            background-color: rgb(39, 50, 56);
            }
QComboBox QAbstractItemView:selected {
             background-color: rgb(69, 75, 84);
            }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1" rowspan="3">
    <widget class="QWidget" name="rigth_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ТАБЛИЦЫ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_profile">
        <property name="text">
         <string>Профиль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_rigel">
        <property name="text">
         <string>Ригель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_user">
        <property name="text">
         <string>Пользователь</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_material">
        <property name="text">
         <string>Материал</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_color">
        <property name="text">
         <string>Цвет</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_customer">
        <property name="text">
         <string>Клиент</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ПОКАЗАТЬ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="r_actual">
           <property name="text">
            <string>Актуальные</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="r_deleted">
           <property name="text">
            <string>Удаленные</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="r_all">
           <property name="text">
            <string>Все</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="top_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/disk.svg</normaloff>icons/disk.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>19</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string>Добавить</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/plus.svg</normaloff>icons/plus.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="table">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
